src/components/Jobs/JobOverviewTab.tsx

```diff
@@ -0,0 +1,85 @@
+import { Anchor, Badge, Card, Text } from '@mantine/core';
+import { format } from 'date-fns';
+import { MdFileDownload } from 'react-icons/md';
+
+import type { Job } from '@/types/job';
+import type { Label } from '@/types/machine';
+import { getJobStateBadgeColor } from '@/utils/job';
+
+type props = {
+  jobData: { job: Job };
+};
+
+function JobOverviewTab({ jobData }: props) {
+  return (
+    <Card my={15} withBorder padding="lg" radius="md">
+      <div className={'absolute right-0 top-0 m-5'}>
+        <Badge
+          variant="light"
+          color={getJobStateBadgeColor(jobData.job.jobMetadata.state)}
+          py={15}
+          radius="md"
+        >
+          {jobData.job.jobMetadata.state}
+        </Badge>
+      </div>
+      <div className={'grid  grid-cols-[15%_85%] gap-y-3'}>
+        <Text fw={700} c="dimmed">
+          Job Id
+        </Text>
+        <Text>{jobData.job.jobMetadata.jobId}</Text>
+
+        <Text fw={700} c="dimmed">
+          Job Label
+        </Text>
+        <div className={'flex flex-row gap-2'}>
+          {jobData.job.jobMetadata.labels.map((label: Label) => (
+            <Badge color={'gray.6'} key={label.name} variant="filled" className="py-3" radius="md">
+              {label.name}
```

Comment:
I think the user will want to be both the label name and the label value somehow. Also this list needs to flex wrap so the content can wrap to a new row if there are too many labels.